src/rebar_coveralls.erl

```diff
@@ -91,11 +106,25 @@ eunit_test_() ->
   Get            = fun(cover_export_enabled, _) -> true end,
   GetLocal       = fun(coveralls_coverdata, _)      -> File;
                       (coveralls_service_name, _)   -> ServiceName;
-                      (coveralls_service_job_id, _) -> ServiceJobId
+                      (coveralls_service_job_id, _) -> ServiceJobId;
+                      (do_coveralls_after_eunit, _) -> true;
```

Comment:
@markusn It needed only for test, i don't know how to not specify mocking parameters. There are default values for rebar at 76 and 77 lines